src/views/Dashboard.vue

```diff
@@ -295,4 +296,11 @@ section header {
 .pending-tooltip {
   max-width: 16.5rem;
 }
+
+#dashboard {
+  & .theme--light.v-btn.v-btn--disabled.v-btn--text, .theme--light.v-btn.v-btn--disabled .v-icon {
+  opacity: 0.4 !important;
+  color: $app-blue !important;
+}
+}
```

Comment:
With this, there will be 3 places where this style is declared.

Do something like this instead:
https://github.com/bcgov/business-create-ui/blob/703aed21cb5c00fc3d4db467e6094c1be5213788/src/assets/styles/overrides.scss#L18

If not, this UI will be even harder to maintain in the future.

Reply:
Please clean up instead of making more special cases.

Reply:
updated